--- a/Application.cpp
+++ b/Application.cpp
@@ -25,8 +25,6 @@
 #include "faust/gui/SoundUI.h"
 #include "faust/misc.h"
 
-
-
 #include "Application.h"
 #include "selectFile.h"
 #include "parametres.h"
--- a/Makefile
+++ b/Makefile
@@ -1,21 +1,18 @@
-LIB ?= /usr/local/lib
-INC = /usr/local/faust
-
 DESTDIR ?=
-PREFIX ?= 
+PREFIX ?=
 
 prefix := $(DESTDIR)$(PREFIX)
 
-all: midiPlayer simplePlayer2 mcMidiPlayer
+all: mcMidiPlayer midiPlayer simplePlayer2
 midiPlayer: simpleLecteur.cpp
-	$(CXX)  -std=c++17 -O3 simpleLecteur.cpp   -ljack -lsndfile -lfaust -llo -I/usr/include/gtk-2.0 -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/cairo -I/usr/include/pango-1.0 -I/usr/lib/x86_64-linux-gnu/gtk-2.0/include -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/atk-1.0 -lgtk-x11-2.0 -lgdk-x11-2.0 -lpangocairo-1.0 -latk-1.0 -lcairo -lgdk_pixbuf-2.0 -lgio-2.0 -lpangoft2-1.0 -lpango-1.0 -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -lpthread `pkg-config  ` -lOSCFaust -o midiPlayer
+	$(CXX)  -std=c++17 -g simpleLecteur.cpp `pkg-config --libs jack` `pkg-config --libs sndfile` -llo `pkg-config --cflags gtk+-2.0` `pkg-config --libs gtk+-2.0` -lfaust -lfaustmachine -lOSCFaust -pthread -o midiPlayer
 
 simplePlayer2: simpleLecteur2.cpp
-	$(CXX)  -std=c++17 -O3 simpleLecteur2.cpp   -ljack -lsndfile -lfaust -llo -I/usr/include/gtk-2.0 -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/cairo -I/usr/include/pango-1.0 -I/usr/lib/x86_64-linux-gnu/gtk-2.0/include -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/atk-1.0 -lgtk-x11-2.0 -lgdk-x11-2.0 -lpangocairo-1.0 -latk-1.0 -lcairo -lgdk_pixbuf-2.0 -lgio-2.0 -lpangoft2-1.0 -lpango-1.0 -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -lpthread `pkg-config  ` -lOSCFaust -o simplePlayer2
+	$(CXX)  -std=c++17 -g simpleLecteur2.cpp `pkg-config --libs jack` `pkg-config --libs sndfile` -llo `pkg-config --cflags gtk+-2.0` `pkg-config --libs gtk+-2.0` -lfaust -lfaustmachine -lOSCFaust -pthread -o simplePlayer2
 
  
 mcMidiPlayer: main.cpp 
-	$(CXX)  -std=c++17 -O3 parametres.cpp parametres.h info.cpp info.h selectFile.cpp selectFile.h Application.cpp main.cpp Application.h   -I$(INC) -lboost_system -lboost_filesystem -ljack -lsndfile -lfaust -lsfml-graphics -lsfml-window -lsfml-system -llo -I/usr/include/gtk-2.0 -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/cairo -I/usr/include/pango-1.0 -I/usr/lib/x86_64-linux-gnu/gtk-2.0/include -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/atk-1.0 -lGL -lGLU  -lGL  -DGL_GLEXT_PROTOTYPES  -lgtk-x11-2.0 -lgdk-x11-2.0 -lpangocairo-1.0 -latk-1.0 -lcairo -lgdk_pixbuf-2.0 -lgio-2.0 -lpangoft2-1.0 -lpango-1.0 -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -lpthread `pkg-config  ` -lOSCFaust -o mcMidiPlayer.bin
+	$(CXX)  -std=c++17 -g parametres.cpp parametres.h info.cpp info.h selectFile.cpp selectFile.h Application.cpp main.cpp Application.h -lboost_system -lboost_filesystem `pkg-config --libs jack` `pkg-config --libs sndfile` `pkg-config --libs sfml-graphics` `pkg-config --libs sfml-window` `pkg-config --libs sfml-system` -llo `pkg-config --cflags gtk+-2.0` `pkg-config --libs glu` -DGL_GLEXT_PROTOTYPES  `pkg-config --libs gtk+-2.0` -lfaust -lfaustmachine -lOSCFaust -pthread -o mcMidiPlayer.bin
 
 
 install: 
@@ -23,17 +20,16 @@ install:
 	chmod 755 $(prefix)/bin/mcMidiPlayer.sh
 	cp mcMidiPlayer.bin  $(prefix)/bin
 	chmod 755 $(prefix)/bin/mcMidiPlayer.bin
-	mkdir $(prefix)/share/mcMidiPlayer 
+	mkdir -p $(prefix)/share/mcMidiPlayer 
 	cp -Rfa Themes $(prefix)/share/mcMidiPlayer
 	cp -Rfa gui $(prefix)/share/mcMidiPlayer
 	cp -Rfa Lang $(prefix)/share/mcMidiPlayer
-	cp midiPlayer $(prefix)/share/mcMidiPlayer
-	chmod 755 $(prefix)/share/mcMidiPlayer
-	cp simplePlayer2 $(prefix)/share/mcMidiPlayer
-	chmod 755 $(prefix)/share/mcMidiPlayer
+	#cp midiPlayer $(prefix)/share/mcMidiPlayer
+	#chmod 755 $(prefix)/share/mcMidiPlayer
+	#cp simplePlayer2 $(prefix)/share/mcMidiPlayer
+	#chmod 755 $(prefix)/share/mcMidiPlayer
 	cp parametres.conf $(prefix)/share/mcMidiPlayer
 
-
 clean:
 	rm -f mcMidiPlayer.bin
 
--- a/mcMidiPlayer.sh
+++ b/mcMidiPlayer.sh
@@ -6,7 +6,7 @@ if [ -d ${dir} ] ; then
 else    
 	mkdir ${dir}
 	echo "Le dossier paramètres ${dir} est créé"
-	cp  /usr/local/share/mcMidiPlayer/parametres.conf ${dir}	
+	cp  /usr/share/mcMidiPlayer/parametres.conf ${dir}	
 fi
 dir="${HOME}/mcMidiPlayer"
 if [ -d ${dir} ] ; then
@@ -15,4 +15,4 @@ else
 	mkdir ${dir}
 	echo "Le dossier de travail ${dir} est créé"
 fi
-/usr/local/bin/mcMidiPlayer.bin ${HOME}
\ No newline at end of file
+/usr/bin/mcMidiPlayer.bin ${HOME}
--- a/simpleLecteur2.cpp
+++ b/simpleLecteur2.cpp
@@ -23,7 +23,7 @@
 #include "faust/gui/SoundUI.h"
 #include "faust/misc.h"
 
- #include <sndfile.h>
+#include <sndfile.h>
 
 using namespace std;
 
